scripts/deno/deno.jsonc

```diff
@@ -4,6 +4,7 @@
 		"lib": ["deno.window"],
 		"strict": true
 	},
+	"unstable": ["temporal"],
```

Comment:
This is [a wonderful API for working with dates](https://tc39.es/proposal-temporal/docs/), but [still behind an unstable flag until Deno 2.0](https://deno.com/blog/v1.40#temporal-api)